File: frontend/runHubFront/src/app/models/result.ts
export interface Result {
    resultId: number;
    userId: string;
    distanceId: number;
    time: string;  // Use string if you handle it as a string initially
    date: string;  // Use string for simplicity in this example
    place: number;
    placeGender: number;
    firstName: string;
    lastName: string;
    club:string;
    gender: string;
    isReadyToShow?: boolean;
}

File: backend/RunHub.Domain/Entities/Result.cs
namespace RunHub.Domain.Entity
{
    public class Result
    {
        public int ResultId { get; set; }
        public string UserId { get; set; }
        public int DistanceId { get; set; }
        public TimeSpan Time { get; set; }
        public int Place { get; set; }
        public int PlaceGender {  get; set; }
        public string Gender { get; set; }
        public AppUser User { get; set; }
        public Distance Distance { get; set; }
    }
}
